### PR TITLE
Add automation for auto-bump Grafana chart

### PR DESCRIPTION
**Pull request by orsenthil**
**What type of PR is this?**

Chore

**What this PR does/ why we need it**:

Bumps Grafana Chart.

**Which issue(s) this PR fixes**:

https://jira.d2iq.com/browse/D2IQ-74158

**Special notes for your reviewer**:

Bumps Grafana Chart.

**Does this PR introduce a user-facing change?**:

```release-note

```

**Checklist**

* [ ] *If a chart is changed, the chart version is correctly incremented.*
* [ ] The commit message explains the changes and why are needed.
* [ ] The code builds and passes lint/style checks locally.
* [ ] The relevant subset of integration tests pass locally.
* [ ] The core changes are covered by tests.
* [ ] The documentation is updated where needed.

### CODE REVIEW

File: Dispatchfile [hunk 1]
```diff
@@ -211,6 +235,8 @@ action(tasks=["test-helm2", "test-helm3"], on=pull_request(chatops=["test"]))
 action(tasks=["test-helm2"], on=pull_request(chatops=["test-helm2"]))
 action(tasks=["test-helm3"], on=pull_request(chatops=["test-helm3"]))
 action(tasks=[do_bump_kommander_catalog_api], on=pull_request(chatops=["bump-catalog-api"]))
+action(tasks=[do_bump_grafana], on=pull_request(chatops=["bump-grafana"]))
 
 # Cron Actions
 action(name="bump-catalog-api", tasks=[do_bump_kommander_catalog_api], on=cron(schedule="0 8 * * 5"))
```
Comment (faiq) [inline Dispatchfile:241]: Can you move this bump to https://github.com/mesosphere/kommander-catalog-api to trigger on tag so we don't have to poll anymore.

**Comment by orsenthil** [comment]
OK, bump-grafana created a PR with the bump - https://github.com/mesosphere/charts/pull/1018

File: Dispatchfile (hunk 1 above)
Comment (orsenthil) [reply Dispatchfile]: https://github.com/mesosphere/kommander-catalog-api/pull/43/

File: Dispatchfile [hunk 2]
```diff
@@ -180,7 +180,7 @@ do_bump_kommander_catalog_api = bump_requirements("bump-catalog-api", git, "stab
   git config --local user.name 'Dispatch CI'
   git config --local user.email '56653984+d2iq-dispatch@users.noreply.github.com'
 
-  bump_requirements --user=mesosphere --project=charts --bump-option=sliced --dependencies=kubeaddons-catalog
+  bump_requirements --user=mesosphere --project=charts --bump-option=sliced --dependencies=grafana
```
Comment (faiq) [inline Dispatchfile:183]:
```suggestion
  bump_requirements --user=mesosphere --project=charts --bump-option=batched --dependencies grafana kubecost
```

**Review by faiq — APPROVED** [review]

**Comment by orsenthil** [comment]
@makkes , @joejulian  - Could you please merge this.

**Comment by orsenthil** [comment]
@makkes , @joejulian - Could you please merge this.

**Review by joejulian — APPROVED** [review]

**Review by makkes — APPROVED** [review]

**Pull request merged**